CHANGELOG.md

```diff
@@ -16,7 +16,11 @@ The evergreen, canonical changelog for *all NILRT branches* can be [found here](
 
 ----
 ## 11.0
-Branch: `nilrt/master/next`
+Branch: `nilrt/25.0/scarthgap`
+
+### nilrt
+#### Added
+- [Added](https://github.com/ni/nilrt/pull/293) a way to handle new fields in the CVE files.
```

Comment:
No need to mention this as this is dev script mainly intended for internal NI use.

Reply:
Fixed.